### PR TITLE
fix: add traefik a record

### PR DESCRIPTION
**Pull request by mfroembgen**

### CODE REVIEW

File: providers.tf [hunk 1]
```diff
@@ -1,4 +1,3 @@
 provider "aws" {
-  alias  = "owner"
-  region = module.this.aws_region
```
Comment (applike-ss) [inline providers.tf:3]: didn't you say something like the region was required when setting an alias or so?

File: route53.tf [hunk 2]
```diff
@@ -45,3 +45,15 @@ resource "aws_route53_record" "default" {
...
+  zone_id = aws_route53_zone.default.zone_id
+  name    = var.domain
+  type    = "A"
+
+  alias {
+    name                   = module.nlb.lb_dns_name
+    zone_id                = module.nlb.lb_zone_id
+    evaluate_target_health = true
+  }
+}
```
Comment (applike-ss) [inline route53.tf:59]: When we have a wildcard record already, why should we make use of this? I'd think there's no need to host something directly on the "root" record of the domain.

Comment (mfroembgen) [reply route53.tf:59]: It's used for traefik dashboard

File: providers.tf (hunk 1 above)
Comment (mfroembgen) [reply providers.tf:3]: Yes, but now you have a different issue as it is duplicated if you define that part by yourself

Comment (applike-ss) [reply providers.tf:3]: Ahh i see.

File: route53.tf (hunk 2 above)
Comment (applike-ss) [reply route53.tf:59]: i know. But why would you want to do that instead of using something like `traefik.${var.domain}`?

Comment (mfroembgen) [reply route53.tf:59]: I want to keep a line with the default traefik config. On the other side, it makes it easy to test for future traefik implementations like tmp alb replacements.

**Pull request closed without merging**